### DISCUSSION TITLE
Pinia-ORM with Axios

### DISCUSSION THREAD

**Discussion by pwfraley** [CodeDredd/pinia-orm#1385] [Q&A] [answered]
Hi all,

I am switching an App from vuex and vuex-orm over to pinia.  Now with vuex-orm there was a Axios Plugin, which allowed you to define loading and updateing functions in the model itself:

User.load()

would call the Load function which uses Axios to load the User from the Backend API.

Now in Pinia-ORM I have not found a plugin like that, and I was wondering what the best practice is for CRUD from the Backend with Pinia-ORM?

Maybe someone can point me to an more full fledge example which also uses a backend api, instead of the standard stuff :)

Thanks :)

**Comment by pallabez** [comment]
Looking for the same. Any update on this 🙏?

**Reply by CodeDredd** [reply, marked as answer]
#1603 😉

**Reply by pallabez** [reply]
That's really awesome. Eager to test it out. 😀
